qiita_pet/handlers/api_proxy/tests/test_sample_template.py

```diff
@@ -21,34 +20,34 @@
 
 @qiita_test_checker()
 class TestSampleAPI(TestCase):
-    info = {
-        "timeseries_type_id": 1,
-        "metadata_complete": True,
-        "mixs_compliant": True,
-        "number_samples_collected": 25,
-        "number_samples_promised": 28,
-        "study_alias": "FCM",
-        "study_description": "DESC",
-        "study_abstract": "ABS",
-        "emp_person_id": qdb.study.StudyPerson(2),
-        "principal_investigator_id": qdb.study.StudyPerson(3),
-        "lab_person_id": qdb.study.StudyPerson(1)
+    def setUp(self):
+        info = {
+            "timeseries_type_id": 1,
+            "metadata_complete": True,
+            "mixs_compliant": True,
+            "number_samples_collected": 25,
+            "number_samples_promised": 28,
+            "study_alias": "FCM",
+            "study_description": "DESC",
+            "study_abstract": "ABS",
+            "emp_person_id": qdb.study.StudyPerson(2),
+            "principal_investigator_id": qdb.study.StudyPerson(3),
+            "lab_person_id": qdb.study.StudyPerson(1)
         }
 
-    new_study = qdb.study.Study.create(
-        qdb.user.User('test@foo.bar'), "Some New Study", [1],
-        info)
+        self.new_study = qdb.study.Study.create(
+            qdb.user.User('test@foo.bar'), "Some New Study", [1],
+            info)
 
-    def setUp(self):
-        fp = join(qiita_config.base_data_dir, 'uploads',
-                  str(self.new_study.id), 'uploaded_file.txt')
+        base_dir = qdb.util.get_mountpoint('uploads')[0][1]
+        fp = join(base_dir, str(self.new_study.id), 'uploaded_file.txt')
```

Comment:
I think it will simplify the handling of the tests if you don't use this test (note code duplication here and in tearDown).
My recommendation would be that you actually create a new file using mkstemp (note that you can indicate the directory in which you want it) and just remove it in the tearDown.